### PR TITLE
feat(Networks firewall): Fixed for DOC-7534

### PR DESCRIPTION
**Pull request by barbnewrelic**
Customer wanted specific procedures on how to configure firewall. Support recommended these general guidelines rather than specific steps because it varies so much.

Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

**Comment by barbnewrelic** [comment]
Temporarily moved PR https://github.com/newrelic/docs-website/pull/5136 to draft so it doesn't accidentally get published before help-neteng reviews it

File: src/content/docs/using-new-relic/cross-product-functions/install-configure/networks.mdx [hunk 1]
```diff
@@ -35,11 +35,11 @@ For future updates to required and supported protocol versions, follow the [`Sec
 
 ## User-facing domains [#user-facing-domains]
 
-Update your allow list to ensure New Relic can communicate with a number of integral domains (listed below). Blocking domains can cause issues with individual product features or prevent pages from loading altogether.
+Update your allow list to ensure New Relic can communicate with a number of integral domains that are listed in this section. Blocking domains can cause issues with individual product features or prevent pages from loading altogether.
 
 This list doesn't cover domains that New Relic connects to that can be blocked without affecting your usage of the product. It also doesn't cover Nerdpacks or other features that communicate with external services that have additional domain requirements.
 
-If your organization uses a firewall that restricts outbound traffic, ensure that the following domains are added to the allow list:
+If your organization uses a firewall that restricts outbound traffic, follow specific procedures for the operating system and the firewall you use to add the following domains to the allow list.
```
Comment (bradleycamacho) [inline src/content/docs/using-new-relic/cross-product-functions/install-configure/networks.mdx:42]:
```suggestion
If your organization uses a firewall that restricts outbound traffic, follow the specific procedures for the operating system and the firewall you use to add the following domains to the allow list.
```

Comment (bradleycamacho) [reply src/content/docs/using-new-relic/cross-product-functions/install-configure/networks.mdx:42]: This is a tiny suggestion, but I felt like "follow specific procedures" implied you this doc would provide them. I think simply adding "the" helps imply that the user has to track those down.

File: src/content/docs/using-new-relic/cross-product-functions/install-configure/networks.mdx [hunk 2]
```diff
@@ -168,7 +168,7 @@ If your organization uses a firewall that restricts outbound traffic, ensure tha
 
 ## APM agents [#agents]
 
-To enhance network performance and [data security](/docs/security/new-relic-security/data-privacy/new-relic-security-protecting-your-data-content), New Relic uses a CDN and DDoS prevention service with a large IP range. New Relic agents require your firewall to allow outgoing connections to the following networks and ports.
+To enhance network performance and [data security](/docs/security/new-relic-security/data-privacy/new-relic-security-protecting-your-data-content), New Relic uses a CDN and DDoS prevention service with a large IP range. New Relic agents require your firewall to allow outgoing connections to the following networks and ports. Follow specific procedures for the operating system and the firewall you use to add the following IP connections to the allow list.
```
Comment (bradleycamacho) [inline src/content/docs/using-new-relic/cross-product-functions/install-configure/networks.mdx:171]:
```suggestion
To enhance network performance and [data security](/docs/security/new-relic-security/data-privacy/new-relic-security-protecting-your-data-content), New Relic uses a CDN and DDoS prevention service with a large IP range. New Relic agents require your firewall to allow outgoing connections to the following networks and ports. Follow the specific procedures for the operating system and the firewall you use to add the following IP connections to the allow list.
```

File: src/content/docs/using-new-relic/cross-product-functions/install-configure/networks.mdx [hunk 3]
```diff
@@ -385,7 +387,7 @@ To configure your firewall to allow synthetic monitors to access your monitored
 
 ## Synthetic monitor private locations [#synthetics-private]
 
-Synthetic private minions report to a specific endpoint based on region. Configure your firewall to allow the private minion to access the endpoint or the static IP addresses associated with the endpoint. These IP addresses may change in the future.
+Synthetic private minions report to a specific endpoint based on region. Follow specific procedures for the operating system and the firewall you use to allow the private minion to access the endpoint or the static IP addresses associated with the endpoint. These IP addresses may change in the future.
```
Comment (bradleycamacho) [inline src/content/docs/using-new-relic/cross-product-functions/install-configure/networks.mdx:390]:
```suggestion
Synthetic private minions report to a specific endpoint based on region. Follow the specific procedures for the operating system and the firewall you use to allow the private minion to access the endpoint or the static IP addresses associated with the endpoint. These IP addresses may change in the future.
```

**Review by bradleycamacho — APPROVED** [review]
Left a few tiny comments here. The flow might only bother me, so feel free to ignore them if you disagree. Merge away if you either fix or don't!

File: src/content/docs/using-new-relic/cross-product-functions/install-configure/networks.mdx (hunk 1 above)
Comment (barbnewrelic) [reply src/content/docs/using-new-relic/cross-product-functions/install-configure/networks.mdx:42]: makes sense, thanks! I will adjust that wording in the other spots too

**Review by bradleycamacho — DISMISSED** [review]

File: src/content/docs/using-new-relic/cross-product-functions/install-configure/networks.mdx [hunk 4]
```diff
@@ -168,7 +168,7 @@ If your organization uses a firewall that restricts outbound traffic, ensure tha
 
 ## APM agents [#agents]
 
-To enhance network performance and [data security](/docs/security/new-relic-security/data-privacy/new-relic-security-protecting-your-data-content), New Relic uses a CDN and DDoS prevention service with a large IP range. New Relic agents require your firewall to allow outgoing connections to the following networks and ports.
+To enhance network performance and [data security](/docs/security/new-relic-security/data-privacy/new-relic-security-protecting-your-data-content), New Relic uses a CDN and DDoS prevention service with a large IP range. New Relic agents require your firewall to allow outgoing connections to the following networks and ports. Follow the specific procedures for the operating system and the firewall you use to add the following IP connections to the allow list.
```
Comment (urbiz-nr) [inline src/content/docs/using-new-relic/cross-product-functions/install-configure/networks.mdx:171]: I'd do "To add the following IP connections to the allow list, follow the specific procedures for the operating system and the firewall you use."

File: src/content/docs/using-new-relic/cross-product-functions/install-configure/networks.mdx [hunk 5]
```diff
@@ -238,6 +238,8 @@ To enhance network performance and [data security](/docs/security/new-relic-secu
   </tbody>
 </table>
 
+### Port 443 recommended [#port-443]
```
Comment (urbiz-nr) [inline src/content/docs/using-new-relic/cross-product-functions/install-configure/networks.mdx:241]: Add header link

File: src/content/docs/using-new-relic/cross-product-functions/install-configure/networks.mdx [hunk 6]
```diff
@@ -385,7 +387,7 @@ To configure your firewall to allow synthetic monitors to access your monitored
 
 ## Synthetic monitor private locations [#synthetics-private]
 
-Synthetic private minions report to a specific endpoint based on region. Configure your firewall to allow the private minion to access the endpoint or the static IP addresses associated with the endpoint. These IP addresses may change in the future.
+Synthetic private minions report to a specific endpoint based on region. Follow the specific procedures for the operating system and the firewall you use to allow the private minion to access the endpoint or the static IP addresses associated with the endpoint. These IP addresses may change in the future.
```
Comment (urbiz-nr) [inline src/content/docs/using-new-relic/cross-product-functions/install-configure/networks.mdx:390]: Same as above.

**Review by urbiz-nr — APPROVED** [review]
Looks good! Left a couple of comments - up to you!

File: src/content/docs/using-new-relic/cross-product-functions/install-configure/networks.mdx (hunk 5 above)
Comment (barbnewrelic) [reply src/content/docs/using-new-relic/cross-product-functions/install-configure/networks.mdx:241]: Not sure what you mean by this. It already has #port-443, I tested it, and it works. Thanks for clarifying!

File: src/content/docs/using-new-relic/cross-product-functions/install-configure/networks.mdx (hunk 4 above)
Comment (barbnewrelic) [reply src/content/docs/using-new-relic/cross-product-functions/install-configure/networks.mdx:171]: yeah i like it that way better, too. thx!

**Pull request merged**